### PR TITLE
Loft uses kw arguments

### PR DESCRIPTION
**Pull request by adamchalmers**
Part of #4600

### CODE REVIEW

File: src/wasm-lib/kcl/src/std/loft.rs [hunk 1]
```diff
@@ -116,28 +105,24 @@ pub async fn loft(exec_state: &mut ExecState, args: Args) -> Result<KclValue, Kc
...
-///     bezApproximateRational = false,
-///     // Tolerance for the loft operation.
-///     tolerance = 0.000001,
-///     // Degree of the interpolation. Must be greater than zero.
-///     // For example, use 2 for quadratic, or 3 for cubic interpolation in
-///     // the V direction. This defaults to 2, if not specified.
-///     vDegree = 2,
-/// })
+/// loft([squareSketch, circleSketch0, circleSketch1],
+///     baseCurveIndex: 0,
```
Comment (adamchalmers) [inline src/wasm-lib/kcl/src/std/loft.rs:109]: Had to remove comments due to https://github.com/KittyCAD/modeling-app/issues/4763

File: src/wasm-lib/kcl/src/std/loft.rs [hunk 2]
```diff
@@ -116,28 +102,31 @@ pub async fn loft(exec_state: &mut ExecState, args: Args) -> Result<KclValue, Kc
...
+///     bezApproximateRational: false,
+///     tolerance: 0.000001,
+///     vDegree: 2,
```
Comment (jtran) [inline src/wasm-lib/kcl/src/std/loft.rs:109]: This needs to get updated after #4770.

**Review by nrc — APPROVED** [review]
lgtm

File: src/wasm-lib/kcl/src/parsing/parser.rs [hunk 3]
```diff
@@ -2539,6 +2539,7 @@ fn fn_call_kw(i: &mut TokenSlice) -> PResult<Node<CallExpressionKw>> {
     let initial_unlabeled_arg = opt((expression, comma, opt(whitespace)).map(|(arg, _, _)| arg)).parse_next(i)?;
     let args = labeled_arguments(i)?;
     ignore_whitespace(i);
+    opt(comma_sep).parse_next(i)?;
```
Comment (jtran) [inline src/wasm-lib/kcl/src/parsing/parser.rs:2542]: ❤️

**Pull request merged**